### DISCUSSION TITLE
关于无法连接上hysteria2服务器，不能正常使用代理。

### DISCUSSION THREAD

**Discussion by ream2006** [apernet/hysteria#1291] [Q&A]
[关于无法连接上hysteria2服务器，不能正常使用代理。.docx](https://github.com/user-attachments/files/18279766/hysteria2.docx)
已在VPS上部署好服务端，但是客户端无法正常使用。初步判断，服务器端无防火墙，其他部署应该也是没错的（日志什么都正常）。客户端我写了五份配置文件，每个都无法正常连接，之后我又换了其他客户端，还是无法正常连接。另外，对客户端的一些配置并不确定，基于此，推测大概率是客户端的配置有问题。所以怎么才可以让客户端正确配置，根据文档的配置似乎不行。
（详情见word文档）

**Comment by haruue** [comment]
请用官方客户端启动 hysteria 的客户端配置文件（你文档里的第 2 个客户端配置文件）， 并提供它的输出。

我们无法对 clash 的报错提供支持。 与 clash 相关的问题请找 clash 社群解决。

**Reply by haruue** [reply]
然后， 实际上的问题， 可能是客户端配置文件里缺少 tls.sni 。
相关的文档请看 https://hysteria.network/docs/advanced/Full-Client-Config/#tls
